radix-engine/src/engine/kernel.rs

```diff
@@ -679,20 +863,20 @@ where
         &Self::current_frame(&self.call_frames).actor
     }
 
-    fn invoke(
+    fn invoke_scrypto(
```

Comment:
Not sure why we're splitting up this logic again after much work went into unifying? We should only be splitting at the points where interpreter should be making the decision.

Reply:
Good point! Will do.